### PR TITLE
workaround for invalid checksum

### PR DESCRIPTION
**Pull request by dasmikko**
I have updated the docs for reverse proxy, with my fix.

### CODE REVIEW

File: website/docs/reverse-proxy.md [hunk 1]
```diff
@@ -57,6 +57,35 @@ Apache virtual server configuration.
     </IfModule>
 ````
 
+## Invalid checksum
+
+Sometimes the gzip compression can mess with the request when running `npm install` and result in error messages like this:
```
Comment (juanpicado) [inline website/docs/reverse-proxy.md:62]: We might link the issue you filled in case someone have similar corner cases.

**Review by juanpicado — APPROVED** [review]
LGTM

File: website/docs/reverse-proxy.md (hunk 1 above)
Comment (dasmikko) [reply website/docs/reverse-proxy.md:62]: Good idea.

**Comment by juanpicado** [comment]
Thanks @dasmikko great contribution.

**Comment by dasmikko** [comment]
You are very welcome! Always happy to help. :)

**Pull request merged**